tei/la-dama-duende.xml

```diff
@@ -2001,6 +2003,7 @@
             <l n="1000">es la novedad, no el miedo,</l>
             <l>que quien admira no teme.</l>
             <stage>Lee.</stage>
+            <p>Con cuidado me tiene vuestra salud, como a quien fue la causa de su riesgo; y así agradecida y lastimada, os suplico, me aviséis della y os sirváis de mí; que para lo uno y lo otro habrá ocasión, dejando la respuesta donde hallasteis esta, advertido, que el secreto importa, porque el día que lo sepa alguno de los amigos, perderé yo el honor y la vida.</p>
```

Comment:
Maybe someone with a better grasp at normalizing the orthography of Calderón's Spanish should check this version.